meilisearch-types/src/tasks.rs

```diff
@@ -383,6 +389,8 @@ impl FromStr for Kind {
             Ok(Kind::IndexCreation)
         } else if kind.eq_ignore_ascii_case("indexUpdate") {
             Ok(Kind::IndexUpdate)
+        } else if kind.eq_ignore_ascii_case("indexSwap") {
+            Ok(Kind::IndexSwap)
```

Comment:
Would it be possible to do a match here? or something to be sure not to forget possible new variants?

Reply:
I don't know if it's possible in fact as it is in the other order...

Reply:
I don't know either :/

Reply:
I am checking [the libraries that exist](https://lib.rs/search?q=from_str+enum) and that could be used as a proc macro to impl that for us instead of forgetting variants.